src/api/lib/tasks/dev/requests.rake

```diff
@@ -199,5 +199,357 @@ namespace :dev do
 
       puts "* Request with delete action #{request.number} has been created."
     end
+
+    desc 'Copy last 10 requests from openSUSE:Factory'
+    task requests_from_open_suse_factory: :development_environment do
+      require 'factory_bot'
+      include FactoryBot::Syntax::Methods
+
+      users = []
+      admin = User.get_default_admin
+      User.session = admin
+
+      # Setup interconnect
+      remote_proj = Project.find_by_name('openSUSE.org')
+      if remote_proj.nil?
+        print_message 'Creating the remote project for interconnect'
+
+        remote_proj = RemoteProject.new(name: 'openSUSE.org',
+                                        remoteurl: 'https://api.opensuse.org/public',
+                                        title: 'Standard OBS instance at build.opensuse.org',
+                                        description: 'This instance delivers the default build targets for OBS.')
+        remote_proj.store
```

Comment:
```suggestion
        remote_proj = RemoteProject.find_or_create_by(name: 'openSUSE.org',
                                        remoteurl: 'https://api.opensuse.org/public')
        remote_proj.store
```

Reply:
I'm intentionally not using `find_or_create_by` because some fields are required and `find_or_create_by` will break if I don't provide all the required fields. In this case this statement is already under a check to find existing remote project.

Reply:
description and title are for sure not required...

Reply:
They are required unfortunately https://github.com/openSUSE/open-build-service/blob/master/src/api/app/models/remote_project.rb#L3

Reply:
Not if you use `Project`...

Reply:
Okay, I've updated this